packages/wdio-cli/src/launcher.js

```diff
@@ -53,6 +50,46 @@ class Launcher {
         this.runnerFailed = 0
     }
 
+    /**
+     * calculates the worker count based on dataprovider injected for each spec file
+     * @return {Number} the worker count required for each capability
+     */
+    calculateWorkerCountForCapability(c) {
```

Comment:
I don't think we need this function as we only use it once

Reply:
I have removed this function.